CodeGen/Generators/UnitsNetGen/QuantityGenerator.cs

```diff
@@ -499,13 +499,13 @@ public static bool TryParse([CanBeNull] string str, [CanBeNull] IFormatProvider
         ///     Parse a unit string.
         /// </summary>
         /// <param name=""str"">String to parse. Typically in the form: {{number}} {{unit}}</param>
+        /// <param name=""provider"">Format to use when parsing number and unit. Defaults to <see cref=""CultureInfo.CurrentUICulture"" /> if null.</param>
         /// <example>
         ///     Length.ParseUnit(""m"", new CultureInfo(""en-US""));
         /// </example>
         /// <exception cref=""ArgumentNullException"">The value of 'str' cannot be null. </exception>
         /// <exception cref=""UnitsNetException"">Error parsing string.</exception>
-        /// <param name=""provider"">Format to use when parsing number and unit. Defaults to <see cref=""CultureInfo.CurrentUICulture"" /> if null.</param>
-        public static {_unitEnumName} ParseUnit(string str, IFormatProvider provider = null)
+        public static {_unitEnumName} ParseUnit(string str, [CanBeNull] IFormatProvider provider)
```

Comment:
The default null parameter was useless (and surprisingly not ambiguous) given there's already a ParseUnit overload that only takes a string.

Reply:
Good catch